scripts/deployChainlinkPricer.js

```diff
@@ -0,0 +1,30 @@
+const yargs = require("yargs");
+
+const ChainlinkPricer = artifacts.require("ChainlinkPricer.sol");
+
+module.exports = async function(callback) {
+    try {
+        const options = yargs
+            .usage("Usage: --network <network> --bot <bot> --asset <asset> --aggregator <aggregator> --oracle <oracle> --gas <gasPrice>")
+            .option("network", { describe: "Network name", type: "string", demandOption: true })
+            .option("bot", { describe: "Bot address", type: "string", demandOption: true })
+            .option("asset", { describe: "Asset address", type: "string", demandOption: true })
+            .option("aggregator", { describe: "Chainlink aggregator address", type: "string", demandOption: true })
+            .option("oracle", { describe: "Oracle module address", type: "string", demandOption: true })
+            .option("gas", { describe: "Gas pricer in GWEI", type: "string", demandOption: false })
```

Comment:
```suggestion
            .option("gas", { describe: "Gas price in GWEI", type: "string", demandOption: false })
```

Also in the sample command, you wrote `--gas 50000000000`, I assume this is wei not gwei?

Reply:
Oh yeah I meant WEI XD